time stamp		# time since system start (microseconds)

uint32 DroneID
float64 speed
geometry_msgs/Point position
geometry_msgs/Quaternion orientation 	#z (yaw): positive north to south (over east); negative: from south to north
					#y (pitch): Airplane coordinate system (nose up +)
					#x (roll):  Airplane coordinate system ( right roll +)



